system_tests/vision.py

```diff
@@ -54,7 +55,15 @@ def tearDownModule():
     bucket_retry(Config.TEST_BUCKET.delete)(force=True)
 
 
-class TestVisionClientLogo(unittest.TestCase):
+class BaseVisionTestCase(unittest.TestCase):
+    def _assert_coordinate(self, coordinate):
+        if coordinate is None:
+            return
+        self.assertIsInstance(coordinate, (int, float))
+        self.assertGreater(abs(coordinate), 0.0)
+
+
+class TestVisionClientLogo(BaseVisionTestCase):
```

Comment:
If this class doesn't use the methods in the base, it doesn't need to inherit from it